code/modules/antagonists/blob/blob_abilities.dm

```diff
@@ -688,6 +688,7 @@
 		B.setMaterial(getInterpolatedMaterial(B.material, reinforcing.material, 0.17))
 		qdel(reinforcing)
 
+		owner.playsound_local(owner.loc, "sound/voice/blob/blob[rand(3, 4)].ogg", 50, 1)
```

Comment:
why are some 3,4 and some 3,7

Reply:
Because 5 6 7 aren't supposed to play

Reply:
what does that even mean

Reply:
Well you see, believe it or not, most sounds have no specific meaning. And I didn't design the sounds for each individual function. Their uses are labeled onto them retroactively.

There are some groups of sounds that I want to use outside of their maybe stated function, but with rand or pick I cant do that. Blob is just a catch all term for stuff that I used for multiple things.

Every rand I did is intentional because i want some things to play and others to not. For whatever reason I thought it needed for some to play 3 and 4 and for them to play 3-7.

Reply:
this PR is to assign them specific meanings, no? otherwise, why not just pick a random sound for every single one of these additions

so, the solution is to name them properly, pick/rand, and not using magic number ranges